src/internal/ClickAwayListener.js

```diff
@@ -21,6 +21,7 @@ export default class ClickAwayListener extends Component {
   };
 
   componentDidMount() {
+    this.isCurrentlyMounted = true;
```

Comment:
I was getting an error by using `isMounted`. I not remember what it was exactly, but I think it was related to it being a read-only property.
I can get you the error message tomorrow if you'd like